manifests/apache.pp

```diff
@@ -52,21 +42,24 @@
       ca            => $default_ca,
       generate      => $generate,
       regenerate    => $regenerate,
-      deploy        => $deploy,
+      deploy        => false,
```

Comment:
I also had a thought of extending `cert` to actually deploy the files directly (without the intermediate RPM step).